js/src/util/sanitizer.js

```diff
@@ -92,8 +92,8 @@ export const DefaultWhitelist = {
 }
 
 export function sanitizeHtml(unsafeHtml, whiteList, sanitizeFn) {
-  if (!unsafeHtml.length) {
-    return unsafeHtml
+  if (!unsafeHtml || !unsafeHtml.length) {
+    return unsafeHtml || ''
```

Comment:
Any specific reason to return an empty string instead of `false`?

Reply:
As there is no JSDoc, I tried to read intent from:
- empty string (String.lenght) returns empty string
- `sanitizeFn` from docs recommends DOMPurify, which returns empty string always (we are using DOMPurify in one product, pretty sure here)
- the return from the function itself, here is the string (`innerHtml`)